src/fswAlgorithms/attGuidance/prescribedRot2DOF/prescribedRot2DOF.c

```diff
@@ -31,14 +35,47 @@ void SelfInit_prescribedRot2DOF(PrescribedRot2DOFConfig *configData, int64_t mod
 }
 
 /*! This method performs a complete reset of the module.  Local module variables that retain
- time varying states between function calls are reset to their default values.
+ time varying states between function calls are reset to their default values. A check is also
+ performed to ensure the user sets the configurable module variables.
  @return void
  @param configData The configuration data associated with the module
  @param callTime [ns] Time the method is called
  @param moduleID The module identifier
 */
 void Reset_prescribedRot2DOF(PrescribedRot2DOFConfig *configData, uint64_t callTime, int64_t moduleID)
 {
+    // Check if the required input messages are linked */
+    if (!HingedRigidBodyMsg_C_isLinked(&configData->spinningBodyRef1InMsg)) {
+        _bskLog(configData->bskLogger, BSK_ERROR, "Error: prescribedRot2DOF.spinningBodyRef1InMsg wasn't connected.");
```

Comment:
:small_red_triangle_down: Remove the `Error: ` string from all these. Calling `_bskLog` with the `BSK_ERROR` parameter will prepend the word "error" in front of these automatically.

Reply:
Done.